subt/script/bridge.bash

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+termtitle() { printf "\033]0;$*\007"; }
+
+( while true; do termtitle "bridge $ROBOT $WORLD"; sleep 5; done ) &
```

Comment:
this should be probably AFTER definition of the ROBOT and WORLD?

Reply:
right - as I have the variables defined, I never noticed, thanks & fixed